### ISSUE TITLE
Workloads listening on sockets do not start

### ISSUE THREAD

**Issue by rvolosatovs** [profianinc/benefice#35]
When running locally on KVM any socket listening configuration makes the job hang forever with no output. For example, adding
```
[[files]]
kind = "listen"
prot = "tcp"
port = 9000
```

to application:
```rust
fn main() {
    println!("Hello, world!");
}
```
compiled as `cargo build --target wasm32-wasi`

produces no output in the log (with `RUST_LOG=info`) and no output in the web terminal window.
Running the workload without the socket produces "Hello, world!" in the web terminal window

**Comment by npmccallum** [comment]
@rvolosatovs This should be very high priority.

**Comment by rvolosatovs** [comment]
@npmccallum we're investigating this now with @haraldh.
Apparently listening sockets do work, but there's no output on the web console.

**Comment by rvolosatovs** [comment]
Seems to be fixed by #36
